plugins/apache-airflow/src/components/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Content,
+  ContentHeader,
+  Header,
+  HeaderLabel,
+  Page,
+  SupportButton,
+} from '@backstage/core-components';
+import { Grid } from '@material-ui/core';
+import React from 'react';
+import { DagTableComponent } from '../DagTableComponent';
+import { StatusComponent } from '../StatusComponent';
+import { VersionComponent } from '../VersionComponent';
+
+export const HomePage = () => (
+  <Page themeId="tool">
+    <Header title="Apache Airflow" subtitle="Workflow management platform">
+      <HeaderLabel label="Owner" value="Colton Padden" />
```

Comment:
Wonder if this should be removed?  Obviously, other companies might not want your name here, but without making this easily customizable such that a company could say "ok who owns my airflow instance", it might not be quite as valuable?